.circleci/config.yml

```diff
@@ -262,6 +262,35 @@ jobs:
             sudo sed -i -re "292s/^(\s+).*\$/\1return _.prepend_to_build_command_raw('')/" /usr/lib/python3/dist-packages/reprotest/build.py
             pytest -vvs tests/test_reproducible_debian_packages.py
 
+  reprepro-update-tor:
+    docker:
+      - image: debian:bullseye
+    steps:
+      - run:
+          name: clone and run reprepro update
+          command: |
+            # TODO: Publish image with these packages pre-installed
+            apt-get update
+            apt-get install -y reprepro ca-certificates dctrl-tools git git-lfs openssh-client
+
+            # Clone the dev repo and configure it
+            git clone git@github.com:freedomofpress/securedrop-dev-packages-lfs.git
```

Comment:
I'm assuming that this will just work and Circle CI already provisions access to the necessary credentials? I didn't see anything explicit in the workstation nightly jobs...